apps/website/app/data/constants.ts

```diff
@@ -0,0 +1 @@
+export const BLOG_PATH = "app/(home)/blog/posts";
```

Comment:
Is separate file for constants a pattern in js/ts world, I remember seeing a similar pattern somewhere else also? In clojure we would put it in utils

Reply:
Moving it to another file solves a type error as well as being used in multiple places.
As for `constants` vs `utils`, just preference I'd think.